### PR TITLE
Form field nightwatch --> wdio

### PR DESCRIPTION
**Pull request by msaikaushik1992**
### Summary
Resolves [this](https://github.com/cerner/terra-core/issues/1302).

### CODE REVIEW

File: dev-site-config/generatedComponentConfig.js [hunk 1]
```diff
@@ -0,0 +1,4132 @@
+/* This file is auto-generated by terra-dev-site's generate-component-config config. */
```
Comment (bjankord) [inline dev-site-config/generatedComponentConfig.js]: this file can be deleted

File: packages/terra-form-field/src/Field.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@
   }
 
   .label {
-    color: var(--terra-form-field-label-color, rgba(0, 0, 0, 0.8));
```
Comment (saedar) [inline packages/terra-form-field/src/Field.scss]: Should the theme changes in this file be part of this work?

File: packages/terra-form-field/src/Field.scss [hunk 3]
```diff
@@ -24,7 +24,7 @@
...
+    color: var(--terra-form-field-label-color, #1c1f21);
```
Comment (JakeLaCombe) [inline packages/terra-form-field/src/Field.scss]: What are these stylistic changes for?

File: packages/terra-form-field/src/terra-dev-site/test/form-field/DefaultField.test.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Base from 'terra-base';
+import Field from '../../../../lib/Field';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+const DefaultField = () => (
+  <Base locale={locale}>
+    <h3> Default Field</h3>
```
Comment (JakeLaCombe) [inline packages/terra-form-field/src/terra-dev-site/test/form-field/DefaultField.test.jsx]: Remove the space before D

File: packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Base from 'terra-base';
-import Field from '../../../Field';
+import Field from '../../../../lib/Field';
```
Comment (JakeLaCombe) [inline packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx]: How come we are using the lib folder? It looks like the other files are using the source file.

File: packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx [hunk 6]
```diff
@@ -35,6 +35,42 @@ const FieldExamples = () => (
...
+      label="Field Label"
+      isInline
+      error="Error Message"
+      isInvalid
+      showOptional
+    >
+      <div style={{ border: 'dashed 1px lightGrey' }}>Control Placeholder</div>
+    </Field>
+
+    <Field
```
Comment (JakeLaCombe) [inline packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx]: What's the diffference between inline-4 and inline-6? They look the same at first glance, which we should use the same if they are.

File: packages/terra-form-field/src/Field.scss (hunk 2 above)
Comment (msaikaushik1992) [reply packages/terra-form-field/src/Field.scss]: Talked to @neilpfeiffer about accessibility issues with help text and error text colors and based on suggestions from UX, had to change the color of label too.

File: packages/terra-form-field/src/Field.scss (hunk 3 above)
Comment (msaikaushik1992) [reply packages/terra-form-field/src/Field.scss]: Talked to @neilpfeiffer about accessibility issues with help text and error text colors and based on suggestions from UX, had to change the color of label too.

File: packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx (hunk 6 above)
Comment (msaikaushik1992) [reply packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx]: [Updated.](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-210f2643e18d38a08a5296e9e1557a38R9)

File: packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx (hunk 5 above)
Comment (msaikaushik1992) [reply packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx]: [Updated](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-f908e7da845ac45c958b8a06974c63c9R1).

File: packages/terra-form-field/src/terra-dev-site/test/form-field/DefaultField.test.jsx (hunk 4 above)
Comment (msaikaushik1992) [reply packages/terra-form-field/src/terra-dev-site/test/form-field/DefaultField.test.jsx]: [Updated.](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-f908e7da845ac45c958b8a06974c63c9R8)

File: packages/terra-form-field/CHANGELOG.md [hunk 7]
```diff
@@ -9,6 +9,9 @@ Unreleased
 ### Changed
 * Updated structure to match terra-dev-site 1.0 and moved doc examples internal.
 
+### Changed
```
Comment (mjhenkes) [inline packages/terra-form-field/CHANGELOG.md]: move this to the unreleased section

**Pull request merged**